### ISSUE TITLE
Complete a final accessibility review of  Result page

### ISSUE THREAD

**Issue by Rizvida1** [CDH-Studio/UpSkill#1447]
**Describe the bug**
Fix accessibility issues with the Result pages.

**Expected behavior**
- [ ]  Axe extensions should show no more errors or structure issues.
- [ ]  App should be compatible with screen readers
